pytest-embedded-idf/tests/test_idf.py

```diff
@@ -556,10 +556,9 @@ def test_unity_test_case_runner(unity_tester):
     ]
 
     one_dev_dut = ['dut-0']
-    multi_dev_duts = ['dut-0', 'dut-1']
 
-    required_names_one_dev = [f'{case_name} [{dut}]' for dut in one_dev_dut for case_name in case_names_one_dev]
-    required_names_multi_dev = [f'{case_name} [{multi_dev_duts[0]}]' for case_name in case_names_multi_dev]
+    required_names_one_dev = [f'{case_name}' for _ in one_dev_dut for case_name in case_names_one_dev]
+    required_names_multi_dev = [f'{case_name}' for case_name in case_names_multi_dev]
 
     junit_case_names = [item.attrib['name'] for item in junit_report]
```

Comment:
could you help rewrite the test case a bit. assert the passed ones and the failed ones with their names.

assert junit_report.nodes[0] == 'normal_case1'
assert junit_report.nodes[1] == 'normal_case2'
assert 'failures' in junit_report.nodes[1]
assert junit_report.nodes[2] == 'multiple_stages_test'
assert junit_report.nodes[3] == 'multiple_devices_test'

Reply:
It seems like 'nodes' is not the right solution. 
```AttributeError: 'xml.etree.ElementTree.Element' object has no attribute 'nodes'```
I will check how to implement it in other way.